packages/server/src/sdk/app/rows/search/internal/sqs.ts

```diff
@@ -425,6 +445,6 @@ export async function search(
     if (err.status === 400 && msg?.match(MISSING_COLUMN_REGEX)) {
       return { rows: [] }
     }
-    throw new Error(`Unable to search by SQL - ${msg}`, { cause: err })
+    throw err
```

Comment:
Think I changed this to make the stack trace nicer in the tests, it'll get put back before merging.